.github/CONTRIBUTING.md

```diff
@@ -104,20 +79,63 @@ To run the link-checker, follow these steps:
 
 ## Contributing to Code
 
-### Code Editor
+### Quick Start Checklist
+
+* Review the [Contributor License Agreement][CLA] requirement.
+* Get familiar with the [PowerShell Repository Git Concepts](../docs/git/README.md).
+* Start a [Github Codespace](#Dev Container) and start exploring the repository.
+* Pick an existing issue to work on! For instance, clarifying a confusing or unclear error message is a great starting point.
+
+### Intro to Git and GitHub
+
+1. Sign up for a [GitHub account](https://github.com/signup/free).
+1. Learning Git and GitHub:
+    - [Git Basics](../docs/git/basics.md): install and getting started
+    - [Good Resources for Learning Git and GitHub][good-git-resources]
+1. The PowerShell repository uses GitHub Flow as the primary branching strategy. [Learn about GitHub Flow](https://guides.github.com/introduction/flow/)
+
+### Code Editing
+
+PowerShell is primarily written in [C#](https://learn.microsoft.com/en-us/dotnet/csharp/tour-of-csharp/overview). While you can use any C# development environment you prefer, [Visual Studio Code (VS Code)][use-vscode-editor] is recommended.
```

Comment:
Please remove language part from link. (https://learn.microsoft.com/**en-us**/dotnet/csharp/tour-of-csharp/overview